src/integrationTests/secureSignal.test.ts

```diff
@@ -185,6 +185,7 @@ describe('Secure Signal Tests', () => {
 
     describe('When SDK initialized after both SDK and SS script loaded - UID2', () => {
       test('should send identity to Google ESP', async () => {
+        window.__uid2 = new UID2();
```

Comment:
Had to clear the UID2, because otherwise this test was receiving the identity from the previous test

Reply:
should we put this in a `beforeEach` block in the case that we want to add more test cases here? I see we already do this on lines 126-127

Reply:
Yep, nice call - i've changed it